aggregator/aggregator.go

```diff
@@ -114,6 +114,16 @@ func (a *Aggregator) tryVerifyBatch(ctx context.Context, ticker *time.Ticker) {
 	}
 	a.compareInputHashes(inputProver, resGetProof)
 
+	// Handle local exit root in the case of the mock prover
+	if resGetProof.Public.PublicInputs.NewLocalExitRoot == "0x17c04c3760510b48c6012742c540a81aba4bca2f78b9d14bfd2f123e2e53ea3e" {
```

Comment:
I'd like to add a config to represent a mock prover, when update this value in the future, it makes us difficult to debug the code